examples/volumes/glusterfs/README.md

```diff
@@ -84,16 +88,12 @@ You can verify that the pod is running:
 $ kubectl get pods
 NAME             READY     STATUS    RESTARTS   AGE
 glusterfs        1/1       Running   0          3m
-
-$ kubectl get pods glusterfs -t '{{.status.hostIP}}{{"\n"}}'
-10.240.169.172
 ```
 
-You may ssh to the host (the hostIP) and run 'mount' to see if the Glusterfs volume is mounted,
+You may run the command `df -h` to see if the GlusterFS volume is mounted correctly:
 
 ```sh
-$ mount | grep kube_vol
-10.240.106.152:kube_vol on /var/lib/kubelet/pods/f164a571-fa68-11e4-ad5c-42010af019b7/volumes/kubernetes.io~glusterfs/glusterfsvol type fuse.glusterfs (rw,relatime,user_id=0,group_id=0,default_permissions,allow_other,max_read=131072)
+$ kubectl exec glusterfs -- df -h
```

Comment:
This command only works if the "df" command is a part of the container image. I believe that's why the prior verification was based off of SSHing to the node.